src/drivers/sd_card/sd_card.h

```diff
@@ -1,6 +1,9 @@
 #ifndef SD_CARD_H
 #define SD_CARD_H
 
+#include "fatfs.h"
+#include "semphr.h"
```

Comment:
These two headers aren't necessary for header declarations, so they should be moved into the .c file